lib/tasks/index.js

```diff
@@ -102,59 +105,85 @@ module.exports = function(ss, router, options) {
-      for(var n in ss.orchestrator.tasks) {
-        delete ss.orchestrator.tasks[n];
-      }
-    },
-    _addTasks: function() {
-      ss.bundler.forEach(function(bundler) {
+        ss.orchestrator.add(bundler.client.name+':pack-css', function(done) {
+          ss.bundler.packAssetSet('css', bundler.client, bundler.toMinifiedCSS,done);
+        });
 
-        // Pack Assets
-        ss.orchestrator.add(bundler.client.name+':pack',function(done) {
-          ss.bundler.pack(bundler.client, function(err) {
-            if (!err) {
-              console.info('Bundler',bundler.client.name,'packed.');
-              bundler.packNeeded = false;
-            }
-            return done(err);
-          }); //TODO bundler.pack();
+        ss.orchestrator.add(bundler.client.name+':pack-js', function(done) {
+          ss.bundler.packAssetSet('js', bundler.client, bundler.toMinifiedJS,done);
+        });
+
+        ss.orchestrator.add(bundler.client.name+':pack-html', function(done) {
+          view(ss, bundler.client, options, function(html) {
+            fs.writeFileSync(bundler.dests.paths.html, html);
+            ss.log.info('✓'.green, 'Created and cached HTML file ' + bundler.dests.relPaths.html);
+
+            //TODO perhaps put this in pack-completion target
+            console.info('Bundler',bundler.client.name,'packed.');
+            bundler.packNeeded = false;
+            done();
+          });
         });
+        ss.orchestrator.add(name+':pack', [
+          name+':pack-needed',
+          name+':pack-css',
+          name+':pack-js',
+          name+':pack-html']);
       });
 
       ss.orchestrator.add('pack-all', this._packTasks(true));
       ss.orchestrator.add('pack-if-needed', this._packTasks());
-      ss.orchestrator.add('pack-report', function() {
+      ss.orchestrator.add('pack-prepare', function() {
         log.info('i'.green, 'Attempting to find pre-packed assets... (force repack with SS_PACK=1)'.grey);
-        ss.bundler.forEach(function(bundler) {
-          if (bundler.packNeeded) {
-            log.info('!'.red, ('Unable to find pre-packed assets for \'' +
-            bundler.client.name + '\'. All assets will be repacked').grey);
-          } else {
-            log.info('✓'.green, ('Serving client \'' + bundler.client.name + '\' using pre-packed assets (ID ' +
-              bundler.client.servingAssetId + ')').grey);
-          }
-        });
+        ss.load();
       });
     },
 
     _packTasks: function(all) {
-      var tasks = ['pack-report'];
+      var tasks = ['pack-prepare'];
       ss.bundler.forEach(function(bundler){
         if (all || bundler.packNeeded) {
           tasks.push(bundler.client.name + ':pack');
+        } else {
+          tasks.push(bundler.client.name + ':pack-unneeded');
         }
       });
       return tasks;
     },
 
+    unload: function() {
+      liveReload.unload();
+    },
+
+    forget: function() {
+      for(var n in ss.orchestrator.tasks) {
```

Comment:
The body of a for in should be wrapped in an if statement to filter unwanted properties from the prototype.